File: Data Structures & Algorithms/merge-sorted-array/submission-0.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int j= 0;
        for(int i= m; i<m+n; i++){
            nums1[i]= nums2[j];
            j++;
        }
        
        for(int k=0; k<(m+n); k++){
            for(int i= k+1; i<(m+n); i++){
                if(nums1[k]>nums1[i]){
                    int temp= nums1[k];
                    nums1[k]= nums1[i];
                    nums1[i]= temp;
                }
            }
        }
    }
}

File: Data Structures & Algorithms/intersection-of-two-arrays/submission-0.cpp
class Solution {
public:
    vector<int> intersection(vector<int>& nums1, vector<int>& nums2) {
        set<int> s;
        for(int num1: nums1){
            for(int num2: nums2){
                if(num1==num2){
                    s.insert(num1);
                }
            }
        }

        return {s.begin(), s.end()};
    }
};